python/packages/autogen-core/src/autogen_core/base/intervention.py

```diff
@@ -14,6 +14,27 @@
 class DropMessage: ...
 
 
+def warn_if_none(result: Any, handler_name: str) -> Any:
```

Comment:
Please name this with a leading underscore to indicate private

Reply:
Sorry for the late reply, I went on a weekend trip. Done.